*** Starting uWSGI 2.0.18 (64bit) on [Fri Mar  1 14:59:57 2019] ***
compiled with version: 4.8.5 20150623 (Red Hat 4.8.5-36) on 01 March 2019 05:47:54
os: Linux-3.10.0-957.5.1.el7.x86_64 #1 SMP Fri Feb 1 14:54:57 UTC 2019
nodename: localhost.localdomain
machine: x86_64
clock source: unix
pcre jit disabled
detected number of CPU cores: 1
current working directory: /home/lyf/test/Django/Django-1.11.20/django/bin/newsite/uwsgi
writing pidfile to uwsgi.pid
detected binary path: /usr/bin/uwsgi
uWSGI running as root, you can use --uid/--gid/--chroot options
*** WARNING: you are running uWSGI as root !!! (use the --uid flag) *** 
chdir() to /home/lyf/test/Django/Django-1.11.20/django/bin/newsite
your processes number limit is 3796
your memory page size is 4096 bytes
detected max file descriptor number: 1024
lock engine: pthread robust mutexes
thunder lock: disabled (you can enable it with --thunder-lock)
probably another instance of uWSGI is running on the same address (:8000).
bind(): Address already in use [core/socket.c line 769]
